src/server/utils.js

```diff
@@ -96,6 +96,14 @@ window.LiveReloadOptions = {
   port: 443,
   https: true,
 };
+</script>`;
+      } else {
+        newbody += `<script>
+window.LiveReloadOptions = { 
+  host: location.hostname, 
+  port: Number(${port}),
+  https: location.protocol,
+};
```

Comment:
why this if?

Reply:
I'm not sure what you mean.  

the `if (process.env.CODESPACES === 'true') {` condition was already there previously.  I only added the `else` condition for all other cases since livereload would not work without it.